package.json

```diff
@@ -25,15 +25,15 @@
   "exports": {
     ".": {
       "types": "./dist/index.d.ts",
-      "require": "./dist/index.js",
-      "import": "./dist/index.mjs",
+      "require": "./dist/index.cjs",
+      "import": "./dist/index.js",
       "source": "./src/index.ts",
       "default": "./dist/index.js"
     },
     "./package.json": "./package.json"
   },
-  "main": "./dist/index.js",
-  "module": "./dist/index.mjs",
+  "main": "./dist/index.cjs",
+  "module": "./dist/index.js",
```

Comment:
do you know why these changes are needed? the previous values match what we have in the monorepo

Reply:
I tried to run the build script with no success, with this error message 
`when I run yarn build I have these errors

[ERROR]  
- package.json with 'type: "module"' - 'exports["."].require' must end with ".cjs"
- package.json with 'type: "module"' - 'exports["."].import' must end with ".js"`
So I followed the error message and fixed the extensions